types/ion.rangeslider/v1/index.d.ts

```diff
@@ -1,52 +0,0 @@
-// Type definitions for for Ion.RangeSlider 1.9.1
```

Comment:
These major-version-specific files should probably be moved over to the new folder.